clang-tools-extra/clang-tidy/kokkos/ImplicitThisCaptureCheck.cpp

```diff
@@ -85,7 +82,6 @@ void ImplicitThisCaptureCheck::check(
         << CE->getDirectCallee()->getName();
     diag(CaptureLocation.getValue(), "the captured variable is used here.",
          DiagnosticIDs::Note);
-    diag(CE->getBeginLoc(), "Kokkos call here.", DiagnosticIDs::Note);
```

Comment:
So that the only real change here, right?

Reply:
There was formatting, and also I changed the string for the regex for `isAllowedPolicy` slightly.